team.go

```diff
@@ -117,3 +141,31 @@ func (api *Client) GetAccessLogs(params AccessLogParameters) ([]Login, *Paging,
 	return response.Logins, &response.Paging, nil
 }
 
+// GetBillableInfoForUser returns the billing_active status of a user by ID.
+func (api *Client) GetBillableInfoForUser(user string) (map[string]BillingActive, error) {
+	values := url.Values{
+		"token": {api.config.token},
+		"user": {user},
+	}
+
+	response, err := billableInfoRequest("team.billableInfo", values, api.debug)
```

Comment:
Instead of
```
+	response, err := billableInfoRequest("team.billableInfo", values, api.debug)
 +	if err != nil {
 +		return nil, err
 +	}
 +	return response.BillableInfo, nil
```

You can just do `return billableInfoRequest("team.billableInfo", values, api.debug)` if you adjust according to the previous comment.